lms/startup.py

```diff
@@ -94,7 +94,7 @@ def enable_stanford_theme():
     theme_root = settings.ENV_ROOT / "themes" / settings.THEME_NAME
 
     # Include the theme's templates in the template search paths
-    settings.TEMPLATE_DIRS.insert(0, theme_root / 'templates')
+    settings.TEMPLATES[0]['DIRS'].insert(0, theme_root / 'templates')
```

Comment:
The `[0]` here is a little fragile, in that if we ever add another template engine (as you recommend in a settings file), then this code will need to change.  Perhaps add `assert len(settings.TEMPLATES) == 1` for a little safety?